pkg/devconsole/proxy/proxy_test.go

```diff
@@ -0,0 +1,59 @@
+package proxy
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProxyEndpoint(t *testing.T) {
+	tests := []struct {
+		testName string
+		method   string
+		expected map[string]interface{}
```

Comment:
You can use your full types here:

```suggestion
		request          ProxyRequest
		expectedResponse ProxyResponse
```